.github/workflows/schedule.py

```diff
@@ -150,8 +154,11 @@ def create_issue(self, title_date: date) -> None:
         title = self.properties['title'] + ' ' + str(title_date)
         flags = []
         repository = self.properties.get('_repository')
-        if repository is not None:
+        requirement_message = 'The defined front-matter keys must be explicitly set'
+        if repository:
             flags.append(f'--repo={repository}')
+        else:
+            require(repository is None, requirement_message, '_repository')
```

Comment:
This is very similar to the body of the nested function you added. You should try to unify all three occurrences under a single mechanism of reuse.